src/containers/committeedescriptions.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import CommitteeDescriptionsComponent from 'components/CommitteeDescriptions';
+
+export default class CommitteeDescriptions extends React.Component {
```

Comment:
Also call the class and anything else like that `Committees`, the name is a bit long